src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see an exampl
+ * in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const newStr = sourceString.replace(/\n/g, '');
+  const objectStyles = {};
+
+  const arrayStyles = newStr.split(';').filter(part => part.length > 2);
+
+  for (let i = 0; i < arrayStyles.length; i++) {
+    const styles = arrayStyles[i].split(':');
+
+    objectStyles[styles[0].trim()] = styles[1].slice(1).trim();
```

Comment:
```suggestion
    objectStyles[styles[0].trim()] = styles[1].trim();
```
If I understood correctly `slice(1)` is doing the same as `trim()` here, so it could be removed